arangod/V8Server/v8-views.cpp

```diff
@@ -42,16 +42,12 @@
 namespace {
 
 ////////////////////////////////////////////////////////////////////////////////
-/// @return the specified object is granted 'level' access
+/// @return the specified vocbase is granted 'level' access
 ////////////////////////////////////////////////////////////////////////////////
-bool canUse(arangodb::auth::Level level, TRI_vocbase_t const& vocbase,
-            std::string const* dataSource = nullptr  // nullptr == validate only vocbase
-) {
+bool canUse(arangodb::auth::Level level, TRI_vocbase_t const& vocbase) {
```

Comment:
is this still needed?

Reply:
yes, this is for LogicalView creation where there is not LogicalView object yet, I haven't figured out a clean way to remove it